### ISSUE TITLE
USER STORY - Hello World Smart Contract and dApp

### ISSUE THREAD

**Issue by russellmorton** [ecadlabs/taqueria#8]
### User Story
As a: intermediate smart contract developer

I want to: Write, deploy and test a "Hello World" Ligo, SmartPy or Michelson smart contract and dapp (in Svelte and React)

So that I can: easily get started with smart contract development.

### Background
"Hello World" application to demonstrate the ability to build deploy and test. Michelson and SmartPy compiling is at a prototype level.

### Use Cases
Intermediate developers wanting to get started with smart contract development.

### Acceptance Criteria
- [x] Smart contract
  - [x] Ligo
  - [x] Michelson
  - [ ] SmartPy
- [x] Able to clone the "Hello World" repository
- [x] Dapp (React)
- [ ] ~Tests for dapp~ Will need to be explored later on. We discussed that it doesn't worth adding these to the simple dapp.
- [x] Tests for smart contract
- [ ] Exploratory testing if completed by SDETs

### Out of Scope
UI testing

### Additional Context and Related Items
Blocked by #9
